src/hello-world/arithmetic.md

```diff
@@ -0,0 +1,39 @@
+---
+minutes: 5
+---
+
+# Arithmetic
+
+```rust,editable
+fn main() {
+    let a: i32 = 12;
+    let b: i32 = 35;
+    let c: i32 = 37;
```

Comment:
Since we've introduced the fallback type, maybe we should use it here to show more idiomatic code.